server/conn.go

```diff
@@ -78,12 +78,14 @@ import (
 	storeerr "github.com/pingcap/tidb/store/driver/error"
 	"github.com/pingcap/tidb/store/tikv/util"
 	"github.com/pingcap/tidb/tablecodec"
+	util2 "github.com/pingcap/tidb/util"
```

Comment:
This seems weird in code, because the name has nothing to do with the real object, and it's very distracting to distinguish between `util/util2`. It's even possible to reverse the name in another file.
I suggest renaming to something like `tikvUtil`/`tidbUtil`.

Reply:
nice catch, done.